tests/bsim/bluetooth/samples/battery_service/prj.conf

```diff
@@ -6,9 +6,10 @@ CONFIG_BT_SMP=y
 CONFIG_BT_BAS=y
 CONFIG_BT_GATT_CLIENT=y
 CONFIG_BT_DEVICE_NAME="bsim_bas"
-CONFIG_BT_ATT_TX_COUNT=5
+CONFIG_BT_ATT_TX_COUNT=10
```

Comment:
Why do we need 10 ATT buffers for BAS? :o

Reply:
This BSIM does kind of stress testing. Here peripheral modifies the BAS Characteristics 5 times a second and there are 2 active connections to the Peripheral. Hence the ATT tx count is increased